contracts/core/Controller.sol

```diff
@@ -274,6 +272,16 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
         _;
     }
 
+    /**
+     * @notice modifier to check if the sender is the liquidation manager address
+     */
+    modifier onlyLiquidationManager() {
```

Comment:
can just make it `require(msg.sender == addressbook.getLiquidationManager(), "C38");`

no need for if statement

Reply:
It relates to the previous comment and also the need for a "toggle"
<img width="590" alt="image" src="https://user-images.githubusercontent.com/99089658/225242101-5cc0c519-6f22-4db4-9cdd-cdb0ce3d35af.png">
<img width="372" alt="image" src="https://user-images.githubusercontent.com/99089658/225242246-13f68ddb-da4b-4c42-bcad-7247f9e15693.png">

Reply:
Also integration test start failing since they were not expecting this statement - would need to adjust integration test if te "if statement" is removed